README.md

```diff
@@ -157,17 +222,27 @@ app.get("/", function(context) {
 });
 ```
 
-See the [Middleware](#middleware) section for more information.
-
 <div id="rocky_authorize"><h3>authorize(<i>callback</i>)</h3></div>
 
-The *authorize()* method allows you to specify a global function to validate or authorize incoming requests. The callback function takes a [Rocky.Context](#context) object as a parameter, and must return either `true` (if the request is authorized) or `false` (if the request is not authorized).
+This method allows you to specify a global function to validate or authorize incoming requests.
+
+#### Parameters ####
+
+| Parameter | Type | Required? | Description |
+| --- | --- | --- | --- |
+| *callback* | Function | Yes | A function to authorize or reject the request |
 
-The *authorize()* method is executed before the main request handler.
+The callback function takes a [Rocky.Context](#context) object as its single argument and must return either `true` (if the request is authorized) or `false` (if the request is not authorized). The callback is executed before the main request handler, so:
 
-- If the callback return `true`, the route handler will be invoked.
+- If the callback returns `true`, the route handler will be invoked.
 - If the callback returns `false`, the [onUnauthorized](#rocky_onAuthorized) response handler is invoked.
```

Comment:
is this the right link? onUnauthorized doesn't appear to match #rocky_onAuthorized